src/main/java/sk/freemap/gpxAnimator/Renderer.java

```diff
@@ -532,6 +532,29 @@ private Point2D drawMarker(final BufferedImage bi, final int frame) {
 		return point;
 	}
 
+	private void drawSimpleCircleOnGraphics2D(Point2D point, Graphics2D g2) {
```

Comment:
When writing new code or refactoring existing code, I would like to see parameters and variables be `final`.